examples/Cargo.toml

```diff
@@ -0,0 +1,36 @@
+[package]
+name = "example"
+description = "Elkodon examples"
+rust-version = { workspace = true }
+version = { workspace = true }
+repository = { workspace = true }
+edition = { workspace = true }
+
+[dependencies]
+elkodon = { workspace = true }
+elkodon_bb_posix = { workspace = true }
+elkodon_bb_container = { workspace = true }
+
+[lib]
+name = "transmission_data"
+path = "src/transmission_data.rs"
+
+[[example]]
+name = "discovery"
+path = "examples/discovery/discovery.rs"
+
+[[example]]
+name = "event_listener"
+path = "examples/event/listener.rs"
+
+[[example]]
+name = "event_notifier"
+path = "examples/event/notifier.rs"
+
+[[example]]
+name = "publish_subscribe_publisher"
+path = "examples/publish_subscribe/publisher.rs"
+
+[[example]]
+name = "publish_subscribe_subscriber"
+path = "examples/publish_subscribe/subscriber.rs"
```

Comment:
Wouldn't it be more elegant to have a structure like before and `transmission_data.rs` in a separate `examples_common` crate in the examples folder?